example/go.mod

```diff
@@ -4,16 +4,13 @@ go 1.20
 
 require (
 	github.com/antihax/optional v1.0.0
-	github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/v5 v5.0.0-00010101000000-000000000000
 	github.com/joho/godotenv v1.5.1
 )
 
 require (
-	github.com/golang/protobuf v1.5.3 // indirect
+	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
-	golang.org/x/oauth2 v0.17.0 // indirect
+	golang.org/x/oauth2 v0.18.0 // indirect
 	google.golang.org/appengine v1.6.8 // indirect
 	google.golang.org/protobuf v1.33.0 // indirect
 )
```

Comment:
Won't it cause the examples to use the published library version rather than local one? 
Overall it's not a bad idea, but we will have to remember to bump version in this file whenever we will do a major release, so it's up to you to decide if we want to leave it or change it.

Reply:
It will always use the local version thanks to `go.work` file that was created some time ago. @TheUnderScorer